alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
```

Comment:
That's because we aren't using a VPC. Our network traffic can't be guaranteed by the infrastructure as being private. With a VPC the system is designed to segment the traffic from all other customers.

If an attacker wanted to view the traffic they could break into the ELB instead of the VPC, so I don't see any increase by adding SSL to the pipeline.